<p><img alt="caee logo" src="images/caee_logo.png" align="left" hspace="10px" vspace="0px"></p>
<p><img alt="iel logo" src="images/iel_logo.png" align="right" hspace="10px" vspace="0px"></p>
<h1 align='center'>Homework 1</h1>

ARE 346P / CE 389H HVAC Design

Professor Dr. Zoltan Nagy

**Due: September 17, 2020**

<h2>Instructions</h2>

Solve each of the following problems and show your work. Make sure to label each answer with the same label as the problem.

You will be graded on both the correct method and solution. Partial credit is awarded for use of the correct equation or method, correct values chosen from tables and charts when required, and the correct answer with appropriate units. 

This assignment includes *five* problems based on Chapter 7 from your textbook.

| PROBLEM | POINTS |
| --- | --- |
| 7.1 | 8 pts |
| 7.21 | 4 pts |
| 7.3 | 4 pts |
| 7.4 | 4 pts |
| 7.27 | 5 pts |
| **TOTAL** | **25 pts** |

## Resources

For this Homework, the following resources might be useful for you:

1. Table A4 from your textbook
2. Psychrometric Chart
    1. [Online Interactive Chart](http://www.flycarpet.net/en/psyonline)
    2. [Downloadable Charts](http://www.handsdownsoftware.com)

<h2>Navigation</h2>

Click on the hyperlink below to go to the problem you are interested in:

- [7.1](#7.1)
- [7.21](#7.21)
- [7.3](#7.3)
- [7.4](#7.4)
- [7.27](#7.27)

<h2>7.1</h2>

Moist air exists at 80$^\circ$F dry-bulb temperature, 0.0150 lbm$_\text{w}$/lbm$_\text{a}$ humidity ratio, and 14.696 psia pressure. Through the use of <u>Table A4 and fundamental relations</u>, determine:

   (a) the dew-point temperature, <br>
   (b) relative humidity, <br>
   (c) volume in ft$^3$/lbm$_a$, and <br>
   (d) the enthalpy in Btu/lbm$_a$ <br>

**Note**: Do not use the psychrometric chart for this problem

<h3>7.1a</h3>

Using <u>Table A4E</u> with $W = 0.0150 \text{ lbm}_\text{w}$:

<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{t}_\text{d} = 68.4 ^\circ \text{F} $
</div>

<h3>7.1b</h3>

Using <u>Table A4E</u> with $ DBT = 80^\circ \text{F} $:
 - $ \text{W}_\text{s} = 0.022340 \text{ lbm}_\text{w}/\text{lbm}_\text{a}$:

By Eq. 7.6 we get:

$$
\begin{align}
\phi &= \frac{ 1 + \frac{0.622}{W_s} }{ 1 + \frac{0.622}{W} } \\
&= \frac{ 1 + \frac{0.622}{0.02234} }{ 1 + \frac{0.622}{0.0150} }
\end{align}
$$


In [1]:
# defining humidity ratios
W = 0.0150
Ws = 0.022340

# solving with eq. 7.6
phi = (1 + (0.622 / Ws)) / (1 + (0.622 / W))

print(f'Relative Humidity: {phi*100}')

Relative Humidity: 67.91782204239041


<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{RH} = 67.9\% $
</div>

<h3>7.1c</h3>

Using <u>Table A4E</u> with $ DBT = 80^\circ \text{F} $:
- $ \text{v}_\text{a} = 13.602 \text{ ft}^3/\text{lbm}_a $
- $ \text{v}_\text{as} = 0.487 \text{ ft}^3/\text{lbm}_a $

By Eq. 7.15 we get: 

$$ 
\begin{align}
v &= v_a + \mu v_{as} \\
&= v_a + \frac{W}{W_s} v_{as} \\
&= 13.602 + \frac{0.0150}{0.02234} \left( 0.487 \right)
\end{align}
$$

In [2]:
# defining specific volumes
va = 13.602
vas = 0.487

# calculating degree of saturation
mu = W / Ws

# solving with eq. 7.15
v = va + mu * vas

print(f'Specific Volume: {v}')

Specific Volume: 13.928991942703671


<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{v} = 13.929 \text{ ft}^3/\text{lbm}_a$
</div>

<h3>7.1d</h3>

Using <u>Table A4E</u> with $ DBT = 80^\circ \text{F} $:
- $ \text{h}_\text{a} = 19.222 \text{ Btu}/\text{lbm}_\text{a} $
- $ \text{h}_\text{as} = 24.479 \text{ Btu}/\text{lbm}_\text{a} $

By Eq. 7.16 we get: 

$$ 
\begin{align}
h &= h_a + \mu h_{as} \\
&= h_a + \frac{W}{W_s} h_{as} \\
&= 19.222 + \frac{0.0150}{0.02234} \left( 24.479 \right)
\end{align}
$$

In [3]:
# defining enthalpies
ha = 19.222
has = 24.479

# solving with eq. 7.16
h = ha + mu * has

print(f'Specific Enthalpy: {h}')

Specific Enthalpy: 35.658213070725154


<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{h} = 36.658 \text{ Btu}/\text{lbm}_a$
</div>

<h2>7.21</h2>

Re-work Problem [7.1](#7.1) again but using the psychrometric chart. Compare your answers to those you found in 7.1. Write out the answers and a brief explanation for each part.

<i>Moist air exists at 80$^\circ$F dry-bulb temperature, 0.0150 lbm$_\text{w}$/lbm$_\text{a}$ humidity ratio, and 14.696 psia pressure.

   (a) the dew-point temperature, <br>
   (b) relative humidity, <br>
   (c) volume in ft$^3$/lbm$_a$, and <br>
   (d) the enthalpy in Btu/lbm$_a$ <br></i>


<h3>7.21a</h3>

Follow line of constant humidity ratio (W) to saturation (moving right to left). Once at saturation, follow vertical line down to the temperature reading.

![7.21a_solution](./images/hw1_7p21a.png)

<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{t}_\text{d} = 68 ^\circ \text{F} $
</div>

<h3>7.21b</h3>

After plotting the initial point, visually interpret between the curved relative humidity lines. 

![7.21b_solution](./images/hw1_7p21b.png)

<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{RH} = 68\% $
</div>

<h3>7.21c</h3>

After plotting the initial point, visually interpret between the slanted, straight lines running from top left to bottom right.

![7.21c_solution](./images/hw1_7p21c.png)

<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{v} = 13.9 \text{ ft}^3/\text{lbm}_a$
</div>

<h3>7.21d</h3>

After plotting the initial point, visually interpret the value using the straight lines running from the top left to the bottom right (they are more flat than the specific volume lines). 

![7.21d_solution](./images/hw1_7p21d.png)

<div class="alert alert-block alert-success">
<b>Solution:</b> $\text{h} = 36.85 \text{ Btu}/\text{lbm}_a$
</div>

<h2>7.3</h2>
    
The atmosphere within a room is at 70$^\circ$F dry-bulb temperature, 50% degree of saturation, and 14.696 psia pressure. The inside surface temperature of the windows is 40$^\circ$F. Will moisture condense out of the air upon the window glass? Show your work and explain your reasoning.

Using <u>Table A4E</u> with $ DBT = 70^\circ \text{F} $:
- $ \text{W}_\text{s} = 0.015832 \text{ lbm}_\text{w}/\text{lbm}_\text{a} $

Using Eq. 7.4, we can determine $W$ because we know that the degree of saturation, $\mu$:

$$
\mu = \frac{W}{W_s}
$$

In [4]:
# defining givens
dbt = 70
mu = 0.5
t_in = 40

# from Table A4E
Ws = 0.015832

# calculating W (eq. 7.4)
W = mu * Ws

print(f'Humidity Ratio: {W}')

Humidity Ratio: 0.007916


Now we can use this value in <u>Table A4E</u> to determine what the dew-point temperature:
- $\text{t}_\text{d} = 51^\circ \text{F} $

In [5]:
t_d = 51
print(f'Indoor Temp = {t_in}:')
if t_in < t_d:
    print('\tCondensation will occur')
else:
    print('\tCondensation will NOT occur')

Indoor Temp = 40:
	Condensation will occur


<div class="alert alert-block alert-success">
<b>Solution:</b> Temperature of the glass is less than the dew-point temperature: <b>condensation occurs</b>
</div>

<h2>7.4</h2>

Assume that the dimensions of the room of Problem [7.3](#7.3) are 30 ft by 15 ft by 8 ft high. Calculate the number of pounds of water vapor in the room.

To solve this problem, we need to determine the partial pressure of water in the space. Once we have that, we can use the ideal gas law to solve for the mass of water.

Using Eq. 7.18, we can determine the partial pressure of water, $P_w$:

$$
P_w = \frac{PW}{0.622 + W}
$$

In [6]:
# defining the givens (all but pressure were defined in problem 3)
P = 14.696

# calculating partial pressure of water
Pw = (P * W) / (0.622 + W)

print(f'Partial Pressure of Water: {Pw}')

Partial Pressure of Water: 0.18468103048660453


Now we can use the idea gas law to calculate the mass of water vapor:

$$
PV = mRT
$$

We need to be coonsistent with our units though, so we will first convert the partial pressure of water to pounds-per-square-foot (psf).

In [7]:
# converting units
Pw_psf = Pw * 12 * 12

# calculating the mass
V = 30 * 15 * 8 # volume of the room
m = (Pw_psf * V) / (85.78 * (70 + 460))

print(f'Mass of Water: {m}')

Mass of Water: 2.105839998861849


<h2>7.27</h2>

In the psychrometric process, moist air at standard atmospheric pressure goes from state 1 at 15$^\circ$C dry-bulb temperature and 10$^\circ$C wet-bulb temperature to state 2 of 35$^\circ$C dry-bulb temperature and 10$^\circ$C dew-point temperature. By using the psychrometric chart, determine:

   (a) the enthalpy-humidity ratio, q’, for the process, and <br>
   (b) the sensible-heat ratio, SHR, for the process

<h3>7.27a</h3>

The psychrometric process is shown below:

![7.27_solution](./images/hw1_7p27.png)

Using the process line, we can draw a line parallel to this and use the protractor to determine $q'$. Or, we can use the following equation to approximate the value:

$$
q' = \frac{\Delta h}{\Delta W}
$$

where the specific enthalpies and humidity ratios for the two states are:

- $h_1 = 29$, $W_1 = 5.6$
- $h_2 = 55$, $W_2 = 7.7$

In [8]:
# defining enthalpies (in kJ per kg of dry air)
h1 = 29
h2 = 55

# defining humidity ratios (g moist air per kg dry air)
W1 = 5.6
W2 = 7.7

qprime = (h2 - h1) / (W2 - W1)

print(f'Enthalpy-Humidity Ratio: {qprime}')

Enthalpy-Humidity Ratio: 12.380952380952378


<div class="alert alert-block alert-success">
<b>Solution:</b> 12 < $q'$ < 14
</div>

<h3>7.27b</h3>

We can use the same parallel line in the protractor to estimate the SHR.

<div class="alert alert-block alert-success">
<b>Solution:</b> SHR = 0.8
</div>